### DISCUSSION TITLE
Flagged as malware

### DISCUSSION THREAD

**Discussion by prizrak2033** [kamaravichow/safe-dot-android#39] [Q&A]
Safedot is being flagged as malware by numerous consumer-facing vendors including MCafee, Malwarebytes, and other vapourware like AVG. How many false positives are you getting/have you been getting?

**Comment by kamaravichow** [comment]
I hope this issue is fixed as I don't see any flag from any kind of provider : https://www.virustotal.com/gui/file/11ba5f95cdb09321dab344f6bb0e135008e2919fed0246baf1e4845d85b097ef?nocache=1

But get it from the official repository only https://github.com/kamaravichow/safe-dot-android/releases/tag/v3.1.0
